va-gov/layouts/home.html

```diff
@@ -87,17 +90,17 @@ <h3 class="hub-links-title"><i class="icon-large fa fa-id-card"></i>Records</h3>
       <div class="usa-grid homepage-benefits-row">
 
         <div class="usa-width-one-third">
-          <h4><a href="/health-care/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small fa fa-medkit"></i>Health Care</a></h4>
+          <h4><a href="/health-care/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small hub-icon-health-care fa fa-medkit hub-background-health-care"></i>Health Care</a></h4>
           <p>Apply for VA health care, find out how to access services, and manage your health and benefits online.</p>
         </div>
 
         <div class="usa-width-one-third">
-          <h4><a href="/disability/" onclick="recordEvent({ event: 'nav-linkslist' });"></a><i class="icon-small fa fa-handshake-o"></i>Disability</a></h4>
+          <h4><a href="/disability/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small fa fa-handshake-o hub-background-disability"></i>Disability</a></h4>
           <p>File a claim for disability compensation for conditions related to your military service, and manage your benefits over time.</p>
         </div>
 
         <div class="usa-width-one-third">
-          <h4><a href="/education/" onclick="recordEvent({ event: 'nav-linkslist' });"></a><i class="icon-small fa fa-book"></i>Education and Training</a></h4>
+          <h4><a href="/education/" onclick="recordEvent({ event: 'nav-linkslist' });"><i class="icon-small fa fa-book hub-background-education"></i>Education and Training</a></h4>
```

Comment:
We had a couple broken links here that are fixed now. Can't believe they made past QA.